clients/lsp-rust.el

```diff
@@ -1289,6 +1290,134 @@ https://github.com/rust-analyzer/rust-analyzer/blob/master/docs/dev/lsp-extensio
   (interactive)
   (lsp-rust-analyzer-move-item "Down"))
 
+
+;; Semantic tokens
+
+;; Modifier faces
+(defvar font-lock-documentation-modifier-face 'font-lock-nop-face
+  "Face name to use for documentation items.")
+
+(defvar font-lock-declaration-modifier-face 'font-lock-nop-face
```

Comment:
does it make sense to move these in `lsp-semantic-tokens`? Also, can you at least prefix them with `lsp-` and if there are rust specific you can prefix with `lsp-rust`

Reply:
I am prefixing them with `lsp-rust-analyzer`.  I see there are already other faces defined in `lsp-rust.el`, such as `lsp-rust-analyzer-inlay-face`.  Perhaps they should stay here, to keep all the rust config in one place.